orttraining/orttraining/python/training/onnxblock/optim/optim.py

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# optim.py
+
+import onnx
+
+import onnxruntime.training.onnxblock._graph_utils as graph_utils
+import onnxruntime.training.onnxblock.building_blocks as building_blocks
```

Comment:
nit: Simplicity: I would consider `from onnxruntime.training.onnxblock import building_blocks`. Here and elsewhere

Reply:
I am not sure if this is a subjective opinion, but I find the `import x.y.z` simpler compared to `from x.y import z`. It also helps in some other areas (such as when globals are defined).
I'm going to leave this one as is and address it in the future if needed.